test/controllers/collection_numbers_controller_test.rb

```diff
@@ -49,7 +49,7 @@ def test_index_observation_id_with_one_collection_number
 
     assert_no_flash
     assert_displayed_title(
-      "Collection Numbers attached to #{obs.unique_format_name.t}".
+      "Collection Numbers for #{obs.unique_format_name.t}".
```

Comment:
Question: Is the following a bad practice?
I've been trying (at least when I have the energy) to use translation symbols as much as possible in test expectations, in order to not break the test if I revise the symbol definition.
(For instance, here I would try to use `query_title_for_observation`.)

Reply:
It is better to use translation symbols.  In this case I didn't fix it since that's the way the old code was.  It also wasn't clear that that was the correct symbol to use.  However, after more hunting I eventually found that it happens here: app/classes/query/modules/titles.rb and that the computed value of the symbol is in fact query_title_for_observation.